contrib/fossil/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "fossil"
+pkgver = "2.23"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = []
+make_check_target = "test"
+# tests require $USER being set
+make_check_env = {"USER": "check"}
+makedepends = ["openssl-devel", "zlib-devel"]
```

Comment:
you probably want `--disable-internal-sqlite` too

Reply:
(otherwise it will have the sqlite+clang powerpc wrong endianness issue and then fail tests later on it, most likely anyway)

Reply:
from the test logs:  

```
Fossil was not compiled with JSON support.
Fossil was not compiled with TH1 docs support.
Fossil was not compiled with TH1 hooks support.
Fossil was not compiled with Tcl support.
```

in alpine config:  
```
		--json \
		--with-th1-docs \
		--with-th1-hooks \
		--with-tcl=1 \
		--with-tcl-private-stubs
```

might want to add those, probably. well, not sure if tcl is useful, but json seems so